torchvision/models/optical_flow/raft.py

```diff
@@ -480,7 +480,16 @@ def forward(self, image1, image2, num_flow_updates: int = 12):
         fmap1, fmap2 = torch.chunk(fmaps, chunks=2, dim=0)
         if fmap1.shape[-2:] != (h // 8, w // 8):
             raise ValueError("The feature encoder should downsample H and W by 8")
-
+        
+        _, _, h_fmap, w_fmap = fmap1.shape
+        if not (((h_fmap // 2**(self.corr_block.num_levels - 1))) < 2) and (
+                ((w_fmap // 2**(self.corr_block.num_levels - 1))) < 2
+        ):
+            min_res = 2 * 2**(self.corr_block.num_levels - 1) * 8
+            raise ValueError(
+                f"input image resolution is too small image resolution should be at least {min_res} (h) and {min_res} (w), got {h} (h) and {w} (w)"
+            )
```

Comment:
It should be equivalent to simply check `h_fmap` and  `w_fmap` against `min_res`? Something like:

```suggestion
        _, _, h_fmap, w_fmap = fmap1.shape
        min_res = 2 * 2**(self.corr_block.num_levels - 1) * 8
        if (h_fmap, w_fmap) < (min_res, min_res):
            raise ValueError(...)
```

Reply:
New version is ready :)

Reply:
This looks good to me, thanks!